hoprd/rest-api/src/session.rs

```diff
@@ -436,8 +433,10 @@ pub(crate) struct SessionClientRequest {
     pub response_buffer: Option<bytesize::ByteSize>,
     /// How many Sessions to pool for clients.
     ///
-    /// If no sessions are pooled, they will be opened ad-hoc when client connects.
-    /// Has no effect for UDP sessions.
+    /// If no sessions are pooled, they will be opened ad-hoc when a client connects.
+    /// It has no effect on UDP sessions in the current implementation.
+    ///
+    /// Currently, the maximum value is 5.
```

Comment:
Please, introduce a `Validator::validate` for the maximum value to enforce it at some point.